src/rest-server/src/config/paiConfig.js

```diff
@@ -32,12 +32,14 @@ try {
 
 let paiConfigData = {
     machineList: paiMachineList,
+    debuggingReservationSeconds: process.env.DEBUGGING_RESERVATION_SECONDS,
 };
 
 
 // define the schema for pai configuration
 const paiConfigSchema = Joi.object().keys({
     machineList: Joi.array(),
+    debuggingReservationSeconds: Joi.number().integer().positive(),
```

Comment:
Shall this validation pass without `Number(process.env.DEBUGGING_RESERVATION_SECONDS)` convertion in line 35?

Reply:
Done